demos/risk/InfrastructureRisk/RoadNetwork/exposure.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<nrml xmlns="http://openquake.org/xmlns/nrml/0.4">
+	<exposureModel id="ex1" category="buildings" taxonomySource="GEM taxonomy">
```

Comment:
@asthapoudel Here we aren't really using the GEM taxonomy to characterize the elements of the network

Reply:
Yes, you are right. In fact, we are not using this. It was just by default there while preparing the exposure file from IPT.